.wp-env.json

```diff
@@ -1,6 +1,7 @@
 {
 	"plugins": [
-		"https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip"
+		"https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip",
+    "https://downloads.wordpress.org/plugin/create-block-theme.latest-stable.zip"
```

Comment:
The spacing here seems off.

Reply:
The `create-block-theme` plugin is not just for testing but for development which is why included it. It will help anyone modifying styles and/or templates to work inside the site editor and export directly to the local directory and then commit to our repo.